pandas/core/groupby/generic.py

```diff
@@ -1750,9 +1755,9 @@ def count(self):
         counted = [
             lib.count_level_2d(x, labels=ids, max_bin=ngroups, axis=1) for x in val
         ]
-        blk = map(make_block, counted, loc)
+        blocks = map(make_block, counted, loc)
```

Comment:
@WillAyd is getting Block/BlockManager stuff out of here still something you're working on?